packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderFirst/en.ftl

```diff
@@ -1,7 +1,7 @@
-verificationReminderFirst-subject = Reminder: Finish creating your account
-verificationReminderFirst-title = Welcome to the { -brand-firefox } family
-verificationReminderFirst-description = A few days ago you created a { -product-firefox-account }, but never confirmed it.
-verificationReminderFirst-sub-description = Confirm now and get technology that fights for and protects your privacy, arms you with practical knowledge, and the respect you deserve.
-confirm-email = Confirm email
-confirm-email-plaintext = { confirm-email }:
-verificationReminderFirst-action = Confirm email
+verificationReminderFirst-subject-2 = Remember to confirm your account
+verificationReminderFirst-title-2 = Welcome to { -brand-firefox }!
+verificationReminderFirst-description-2 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account within the next 15 days or it will be automatically deleted.
+verificationReminderFirst-sub-description-2 = Don’t miss out on tech that puts you and your privacy first.
+confirm-email-2 = Confirm account
+confirm-email-plaintext-2 = { confirm-email-2 }:
```

Comment:
@bcolsson does `:` mean the same thing in all languages when used this way?

Reply:
Hm....I just did a search and this is the only reference to `confirm-email-2` in the `verificationReminderFirst` directory. I'm not aware of a reason why we couldn't just include the `:` in the string (which could solve this issue)

Reply:
@chenba 
Great question, thanks for asking!
Typically, yes, although different languages may need to adjust the structure slightly. For example French might need to insert a non breaking space before the colon. Even if the meaning were different since the colon is part of the string (and thus can be changed when translated), localizers are able to adjust the string to work in their languages.
Also of note, that a similar string structure already exists out in the wild (e.g. verifyPrimary-action-plaintext) so we should be safe.

Reply:
hmm I don't think I'm reading this correctly.  The localiser can still use another symbol (and in a different location relative to `confirm-email-2`); they just can't break up `confirm-email-2` or translate it differently.  So I think we are ok here?

Reply:
Thanks, Bryan.  (Saw your response after I posted my above comment.)